packages/playground/php-cors-proxy/cors-proxy.php

```diff
@@ -15,6 +15,18 @@
 $server_host = $_SERVER['HTTP_HOST'] ?? '';
 $origin = $_SERVER['HTTP_ORIGIN'] ?? '';
 
+// Define allowlist of Content-Types
+$allowed_content_types = [
+    'application/json',
```

Comment:
There's also non-standard `text/json` and `text/javascript`. Also, there are image content types, font content types, I wonder what else